.circleci/config.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+jobs:
+  build:
+    parallelism: 1
+    working_directory: ~/ezcater
+    docker:
+      - image: circleci/ruby:2.4
```

Comment:
Can we start with 2.3.3 since that is what we are targeting with ez-rails right now? That may quickly switch to 2.4.1 once we put ez-rails support on a 0.49-release branch.

Reply:
yeah - we target different ruby versions in different apps. SemTools is on 2.4, so that is my bad. Will update to ezrails version for now....

Reply:
Or do we think it's worth running this stuff on multiple ruby instances for the versions we currently target?

Reply:
Does circle provide a simple way to do that? Since this is public, I'd ultimately rather get this switched to Travis where testing against multiple versions is easily supported. In fact one easy way to do it here might be to run [wwtd](https://github.com/grosser/wwtd)